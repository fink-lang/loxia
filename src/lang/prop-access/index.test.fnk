{fink2js} = import '../../testing'
{describe, it, eq, snapshot} = import '../../testing/jest'


describe:: 'member', fn:
  it:: 'compiles', fn:

    fink2js(`
      foo = spam.shrub
      computed_member = item.'bar spam'
      computed_member2 = item.(Symbol.iterator)
      reserved_prop = item.arguments
    `) eq snapshot

