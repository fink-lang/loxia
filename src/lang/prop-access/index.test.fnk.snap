// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`member compiles 1`] = `
"export const foo = spam.shrub;
export const computed_member = item[\`bar spam\`];
export const computed_member2 = item[Symbol.iterator];
export const reserved_prop = item.arguments;"
`;
